src/AccessibilityInsights.SharedUx/Dialogs/ContainedDialog.cs

```diff
@@ -65,6 +65,11 @@ public Task<bool> ShowDialog(Action<ContainedDialog> hideDialog)
             return Task.Run((Func<bool>)WaitAndHide);
         }
 
+        public void Show()
```

Comment:
intentional to provide a imperative show/hide experience rather than a blocking showdialog where the dialog is in charge of executing the operation